src/argilla_server/models/questions.py

```diff
@@ -151,13 +154,57 @@ def check_response(self, response: ResponseValue, status: Optional[ResponseStatu
             raise ValueError("This Ranking question expects a list of unique values, but duplicates were found")
 
 
+class SpanQuestionResponseValueItem(BaseModel):
+    label: str
+    start: int = Field(..., ge=0)
+    end: int = Field(..., ge=1)
+
+    @root_validator(skip_on_failure=True)
+    def check_start_and_end(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+        start, end = values.get("start"), values.get("end")
+
+        if start is not None and end is not None and end <= start:
+            raise ValueError("Span question response value 'end' must have a value greater than 'start'")
+
+        return values
+
+
+class SpanQuestionResponseValue(BaseModel):
+    value: List[SpanQuestionResponseValueItem] = Field(..., max_items=SPAN_QUESTION_RESPONSE_VALUE_MAX_ITEMS)
+
+
+class SpanQuestionSettings(BaseQuestionSettings):
+    type: Literal[QuestionType.span]
+    options: List[ValueTextQuestionSettingsOption]
```

Comment:
```suggestion
    options: List[ValueTextQuestionSettingsOption]
    field: str # <- ??
```

Reply:
I'm not using it to check responses that's why I didn't add it. Additionally the handler schemas are already checking that it's there and we are validating that the field is valid in the question creation.